### PR TITLE
fix: Read embedder URL from env

### PR DESCRIPTION
**Pull request by Dschoordsch**
I had hardcoded it before 🤦

## Demo

[If possible, please include a screenshot or gif/video, it'll make it easier for reviewers to understand the scope of the changes and how the change is supposed to work. If you're introducing something new or changing the existing patterns, please share a Loom and explain what decisions you've made and under what circumstances]

## Testing scenarios

[Please list all the testing scenarios a reviewer has to check before approving the PR]

- [ ] Scenario A
  - Step 1
  - Step 2...

- [ ] Scenario B
  - Step 1
  - Step 2....

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label https://github.com/ParabolInc/parabol/labels/Skip%20Maintainer%20Review if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/server/graphql/public/types/User.ts [hunk 1]
```diff
@@ -28,6 +28,20 @@ import {UserResolvers} from '../resolverTypes'
 
 declare const __PRODUCTION__: string
 
+const MODEL = 'Embeddings_ember_1'
+const EMBED_URL = (() => {
+  try {
+    const availableModels =
+      process.env.AI_EMBEDDING_MODELS && JSON.parse(process.env.AI_EMBEDDING_MODELS)
+    return availableModels.find(
+      ({model}: {model?: string}) => model?.split(':')[1] === 'llmrails/ember-v1'
```
Comment (rafaelromcar-parabol) [inline packages/server/graphql/public/types/User.ts:37]: @Dschoordsch why is `llmrails/ember-v1` hardcoded? That's just a possibility but in the future we might have other thing there. I might be lacking of JS knowledge to read this expression 😺

Comment (Dschoordsch) [reply packages/server/graphql/public/types/User.ts:37]: You're right, it's hardcoded. It's also hardcoded here: https://github.com/ParabolInc/parabol/blob/46770a048bfd4cd21c0e1cab89abdc01f8a79af3/packages/embedder/ai_models/TextEmbeddingsInference.ts#L14

I tried to strike the balance between getting stuff out now vs. making it future proof. Even if we add more models, we would need to test which one to use here and adjust accordingly. Having it configurable via environment for example add some complexity which might never be needed.

Comment (rafaelromcar-parabol) [reply packages/server/graphql/public/types/User.ts:37]: If I'm not mistaking, on that file TextEmbeddingsInference.ts you have that model but there is [another one there too](https://github.com/ParabolInc/parabol/blob/46770a048bfd4cd21c0e1cab89abdc01f8a79af3/packages/embedder/ai_models/TextEmbeddingsInference.ts#L8).

Comment (rafaelromcar-parabol) [reply packages/server/graphql/public/types/User.ts:37]: If I'm not mistaking, on that file TextEmbeddingsInference.ts you have that model but there is [another one there too](https://github.com/ParabolInc/parabol/blob/46770a048bfd4cd21c0e1cab89abdc01f8a79af3/packages/embedder/ai_models/TextEmbeddingsInference.ts#L8).

**Pull request merged**